v2.1/inverted-indexes.md

```diff
@@ -156,6 +158,14 @@ Now, run a query that filters on the `JSONB` column:
 +--------------------------------------+----------------------------------+--------------------------------------------------------------------------+
 (2 rows)
 ~~~
+### Comparisons
+Currently, inverted indexes only support equality. If you require comparisons like `>`, `<=`, etc., you can create an index on a computed column using your json payload, and then create a regular index on that. So if you wanted to write a query where the value of "foo" is greater than three, you would:
+1) Create your computed column:
```

Comment:
The list of steps is rendering a little weird on the built page at http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/953ab54c7314a112c6b0be7ec78ee6dd8743b3c7/dev/inverted-indexes.html

1. Suggest editing slightly to:

```
... you would:

1. Create your computed column: `CREATE TABLE test(id INT, data JSONB, foo INT AS ((data->>'foo')::int) STORED);`
2. Create an index on your stored column: `CREATE INDEX test_idx ON test (foo);`
3. Execute your query: `SELECT * FROM test3 WHERE foo > 3;`
```

2. I don't see where `test3` is created before step 3; do you mean `SELECT * FROM test_idx` since `test_idx` was created in step 2?

Reply:
Test 3 was an artifact from Ben's example. Good catch. Using the test table should use test_idx implicitly.